packages/@dcl/inspector/src/components/EntityInspector/NftShapeInspector/utils.ts

```diff
@@ -124,3 +155,26 @@ export const NFT_STYLES = [
     label: 'None'
   }
 ]
+
+export type UrnTokens = {
+  network?: number
+  contract?: string
+  token?: string
+}
+
+export const DEFAULT_NETWORK = NETWORKS[0]
+
+export function getUrn({ network, contract, token }: UrnTokens): string {
+  const _network = NETWORKS.find(($) => $.value === network)?.label || DEFAULT_NETWORK.label
+  return `urn:decentraland:${_network}:${contract}:${token}`
```

Comment:
this is missing the contract standard. I don't know if we support multiple, if so we should add a dropdown to choose one (either erc721 or erc1155) or if we only support erc721 we could simply add it to the template string here.